src/Dfe.PlanTech.Domain/Users/Interfaces/IGetAllAnswersForLatestSubmissionQuery.cs

```diff
@@ -0,0 +1,8 @@
+using Dfe.PlanTech.Domain.Answers.Models;
+
+namespace Dfe.PlanTech.Domain.Users.Interfaces;
+
+public interface IGetAllAnswersForLatestSubmissionQuery
```

Comment:
Naming conventions; we have 

`IGetRecommendationIntro`
`IGetRecommendationChunks`
`IGetSubTopicRecommendation`

but also

`IGetAllAnswersForLatestSubmissionQuery`.

Would be good if they all end with query, or none of them have it.

On a related note,

One returns an IEnumerable, one returns a List. Again, would be good to keep consistent.